### ISSUE TITLE
Storybook enhancements/fixes

### ISSUE THREAD

**Issue by agliga** [eBay/ebayui-core#1549]
# Feature Request

## Description
Several stories have issues or other things that need to be fixed or enhanced.

## Context
(Not fully extensive)
* Add spread attributes for most stories
* Rename standard story
* Fix `a11y` issues and bugs. Made default behavior not have any accessibility failures
* Add code examples for all stories and make sure they work properly

**Comment by agliga** [comment]
Fixed by Goutham in 8.2.0
